setup.py

```diff
@@ -12,7 +12,7 @@ def readme():
 extras_require = {
     "prophet": [
         "fbprophet>=0.5, <0.7",
-        "holidays==0.9.11",
+        "holidays==0.16",
```

Comment:
Why do we need to pin to `0.16`? Can defer to `holidays>=0.14.2` set by `prophet` [here](https://github.com/facebook/prophet/blob/main/python/requirements.txt)?

Reply:
Sorry @jklaise just realised this is a dependabot PR not yours! The question still stands though, can we just remove `holidays` from our `setup.py` now?

Reply:
In fact, ignore me! This is what you've done in https://github.com/SeldonIO/alibi-detect/pull/627. We can probably just close this PR...